src/resources/views/crud/fields/inc/translatable_icon.blade.php

```diff
@@ -1,3 +1,15 @@
-@if (method_exists($crud->model, 'isTranslatableAttribute') && ($crud->model->isTranslatableAttribute($field['name']) || (isset($field['store_in']) ? $crud->model->isTranslatableAttribute($field['store_in']) : false)) && config('backpack.crud.show_translatable_field_icon'))
+@php
+    //if field name is array we check if any of the arrayed fields is translatable
+    $translatable = false;
+    if($crud->model->translationEnabled()) {
+        foreach((array) $field['name'] as $field_name){
+            if($crud->model->isTranslatableAttribute($field_name)) {
+                $translatable = true;
+            }
+        }
+    }
+
+@endphp
+@if ($translatable || (isset($field['store_in']) ? ($crud->model->translationEnabled() ? $crud->model->isTranslatableAttribute($field['store_in']) : false) : false) && config('backpack.crud.show_translatable_field_icon'))
```

Comment:
There is a mistake on this condition, now, it is something like this;
`if( translatable || store_in_is_translatable && show_translatable_icon)`
It's missing parenteses on the first two conditions, otherwise even though user doesn't want to show translatable icon on configs, it will show, since `true || false && false` evaluates to true.

What I recommend, to make everything easy to read, is to move the `store_in_is_translatable` condition group inside the first if above.

```
    if($crud->model->translationEnabled()) {
        foreach ...

        if(isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in'])) {
            $translatable = true;
        }
    }
```

And then, 
```suggestion
@if ($translatable && config('backpack.crud.show_translatable_field_icon'))
```

Reply:
Thanks for spotting @promatik 

Agree with your proposed solution. 

Just submitted the fix. 

Best,
Pedro